internal/daysteps/daysteps.go

```diff
@@ -12,9 +17,49 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	var steps int
+	var duration time.Duration
+	var err error
+	entries := strings.Split(data, ",")
+	if len(entries) != 2 {
+		return 0, 0, fmt.Errorf("неверный формат")
+	}
+	steps, err = strconv.Atoi(entries[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать ошибки всегда
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```